packages/ui-dom-utils/src/findDOMNode.ts

```diff
@@ -81,8 +82,14 @@ function findDOMNode(el?: UIElement): Element | Node | Window | undefined {
       ? (node as any).constructor.componentId
       : (node as any).constructor.name
     console.error(
-      `Error: ${elName} doesn't have "ref" property.\nReactDOM.findDOMNode is removed in React 19, consider using refs instead. From InstUI v9, components must have the "ref" property for findDOMNode to work.\nSee more here: https://instructure.design/#accessing-the-dom`
+      `Error: ${elName} doesn't have "ref" property.\n
+      your code will likely not work with React 19+, because ReactDOM.findDOMNode() was removed.\n
+      See more here: https://instructure.design/#accessing-the-dom`
     )
+    const ReactDomInstance = ReactDOM as any
+    if (typeof ReactDomInstance.findDOMNode === 'function') {
+      return (ReactDOM as any).findDOMNode(node)
+    }
```

Comment:
note: I have not tested this with React 18, we will need to test the whole RC with it (and with React 17)